event_lead_event/models/event.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2018 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import _
+from odoo import api, fields, models
+from odoo.exceptions import ValidationError
+
+
+class Event(models.Model):
+
+    _inherit = "event.event"
+
+    lead_event = fields.Boolean(
+        string='Lead Event',
+        index=True,
+    )
+
+    @api.constrains('lead_event')
+    def check_only_one_leadevent(self):
+        self.ensure_one()
```

Comment:
I'm not sure about this. If someone does `events.write({'lead_event': True})` where `len(events) > 1` won't this method be called with self being a recordset of multiple records, thus throwing an error on `ensure_one()` ?
A test on that could be good to ensure a ValidationError is raised.